cosmos/__init__.py

```diff
@@ -40,6 +40,36 @@
 
 logger = get_logger(__name__)
 
+try:
+    from cosmos.operators.gcp_cloud_run_job import (
+        DbtLSGcpCloudRunJobOperator,
```

Comment:
I just noticed we are not importing the Build operator from other similar modules, apparently, except for `DbtBuildLocalOperator`. We can start correcting this by adding here:

```suggestion
        DbtBuildGcpCloudRunJobOperator,
        DbtLSGcpCloudRunJobOperator,
```